src/dotnet/commands/dotnet-restore/LocalizableStrings.resx

```diff
@@ -172,4 +172,19 @@ This is equivalent to deleting project.assets.json.</value>
   <data name="CmdInteractiveRestoreOptionDescription" xml:space="preserve">
     <value>Allows the command to stop and wait for user input or action (for example to complete authentication).</value>
   </data>
+  <data name="CmdLockedModeOptionDescription" xml:space="preserve">
+    <value>Do not allow updating project lock file.</value>
+  </data>
+  <data name="CmdLockFilePathOption" xml:space="preserve">
+    <value>ProjectLockFile_Path</value>
+  </data>
+  <data name="CmdLockFilePathOptionDescription" xml:space="preserve">
+    <value>Project lock file path to be used to write lock file.</value>
```

Comment:
File path where the lock file will be written? The way it is seems strange. cc @KathleenDollard

Reply:
"Output location where project lock file is written"

Reply:
How about `Output location where project lock file is written. By default, this is '<PROJECT_ROOT>\packages.lock.json'.`